### DISCUSSION TITLE
PowershellGet 3 backward compatibility with Powershell 5.1 modules

### DISCUSSION THREAD

**Discussion by denisky** [PowerShell/PSResourceGet#988] [Q&A]
Does PowershellGet 3 supports Powershell 5.1 modules?

I'm using PowershellGet 3 to publish and install powershell modules.
My module is supporting only Powershell 5.1 at the moment.

When installing a module using `Install-PSResource -Scope CurrentUser` the module gets installed under `$home\Documents\PowerShell\Modules` as documented.

My concern is that the location is not present by default in `$env:PSModulePath` when using Powershell 5.1.
``` pwsh
PS C:\> $env:PSModulePath -split ';'
C:\Users\myuser\OneDrive - Genetec Inc\Documents\WindowsPowerShell\Modules
C:\Program Files\WindowsPowerShell\Modules
C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules
```

Are we intended to add the path to the environment variable if we want to use the module in Powershell 5.1?

**Comment by jdhitsolutions** [comment]
I'm testing v3.0.19 on Windows PowerShell and installing to CurrentUser uses `documents\WindowsPowerShell\Modules.`

**Comment by denisky** [comment]
Turns out I have the issue only in VSCode Terminal.
I believe the powershell extensions is not configured properly.

If I test in Windows Powershell it work as expected.
So it's all good.

Thanks
